javalib/src/main/scala/java/lang/process/GenericProcess.scala

```diff
@@ -161,7 +161,7 @@ private object GenericProcess {
             watchedProcesses.forEach { (completion, handle) =>
               if (completion.isCancelled())
                 watchedProcesses.remove(completion)
-              else if (handle.process.waitFor(1, TimeUnit.MILLISECONDS)) {
+              else if (handle.process.waitFor(5, TimeUnit.MILLISECONDS)) {
```

Comment:
I concur that this is likely to improve the odds without introducing prohibitive cost
to `waitFor()`/`isAlive()` in other threads.